frontend/src/components/UploadImageCard.tsx

```diff
@@ -29,16 +29,12 @@ const getCookie = (name: String) => {
 function UploadImagePage(): JSX.Element {
   const [imageUrl, setImageUrl] = useState("");
   const [isLoading, setIsLoading] = useState(false);
+  const [isDragging, setIsDragging] = useState(false);
   const imageInput = useRef<HTMLInputElement>(null);
 
-  const handleImageUpload = async (event: ChangeEvent<HTMLInputElement>) => {
+  const handleImageUpload = async (file: File) => {
```

Comment:
Just a quick refactoring since the way file upload via drop works differently than upload via clicks